proxy/src/deshred.rs

```diff
@@ -0,0 +1,194 @@
+use itertools::Itertools;
```

Comment:
need to run `cargo +nightly fmt` at the root of the proj

Reply:
Fixed in https://github.com/jito-labs/shredstream-proxy/pull/40/commits/9a21c9157006c7731a6a6dac16d2da8770e29c10